src/attributes/classes.js

```diff
@@ -158,12 +156,13 @@ jQuery.fn.extend({
 	},
 
 	hasClass: function( selector ) {
-		var className = " " + selector + " ",
-			i = 0,
-			l = this.length;
-		for ( ; i < l; i++ ) {
-			if ( this[i].nodeType === 1 &&
-				( " " + getClass( this[i] ) + " " ).replace( rclass, " " )
+		var className, elem,
+			i = 0;
+
+		className = " " + selector + " ";
```

Comment:
> If I do that jscs (after doing $> grunt) tell me:

Old config, we will update jscs soon enough. Ignore that for a minute. So what is the size diff without `className` initialization?